### PR TITLE
adding downtown1

### PR DESCRIPTION
**Pull request by ThomasLipp**
This is a single card just to make sure that I'm not doing anything you find objectionable

### CODE REVIEW

File: eldritch/test_encounters.py [hunk 1]
```diff
@@ -435,3 +435,45 @@ def testAdministration7Fail(self):
...
+    self.assertEqual(self.char.clues, 2)    
+
+  def testAsylum1Three(self):
+    self.char.lore_luck_slider = 2
+    self.state.event_stack.append(encounters.Asylum1(self.char))
+    with mock.patch.object(events.random,"randint", new=mock.MagicMock(return_value=5)):
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 3)
+    self.assertEqual(self.char.clues, 3)
+
```
Comment (fingoldfish) [inline eldritch/test_encounters.py:466]: A few style nitpicks here - I recommend two blank lines between classes, and a newline at the end of the file.

File: eldritch/encounters.py [hunk 2]
```diff
@@ -136,10 +136,27 @@ def Administration7(char):
   gain = events.Gain(char, {"dollars": 8})
   arrest = events.Arrested(char)
   return events.PassFail(char, check, gain, arrest)
+def Asylum1(char):
+  check = events.Check(char, "lore", 0)
+  loss = events.Loss(char, {"sanity": 1})
+  gain1 = events.Gain(char, {"clues": 1})
```
Comment (fingoldfish) [inline eldritch/encounters.py:142]: I recommend making this a single GainOrLoss event (which supports doing gains and losses at the same time). This makes the execution of this event atomic, which matters if the sanity loss drives the player insane.
(If you have a sequence of loss and then gain, but the loss drives the player insane, then the gain won't be executed.)

File: eldritch/test_encounters.py [hunk 3]
```diff
@@ -435,3 +435,45 @@ def testAdministration7Fail(self):
...
+class AsylumTest(EncounterTest):
+
+  def setUp(self):
+    super(AsylumTest, self).setUp()
+    self.char.place = self.state.places["Asylum"]
+
+  def testAsylum1Zero(self):
+    self.state.event_stack.append(encounters.Asylum1(self.char))
+    with mock.patch.object(events.random,"randint", new=mock.MagicMock(return_value=3)):
```
Comment (fingoldfish) [inline eldritch/test_encounters.py:447]:
```suggestion
    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=3)):
```

File: eldritch/test_encounters.py [hunk 4]
```diff
@@ -435,3 +435,45 @@ def testAdministration7Fail(self):
...
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 2)
+    self.assertEqual(self.char.clues, 1)
+
+  def testAsylum1One(self):
+    self.state.event_stack.append(encounters.Asylum1(self.char))
+    with mock.patch.object(events.random,"randint", new=mock.MagicMock(return_value=5)):
```
Comment (fingoldfish) [inline eldritch/test_encounters.py:454]:
```suggestion
    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
```

File: eldritch/test_encounters.py [hunk 5]
```diff
@@ -435,3 +435,45 @@ def testAdministration7Fail(self):
...
+      self.resolve_until_done()
+    self.assertEqual(self.char.sanity, 3)
+    self.assertEqual(self.char.clues, 2)    
+
+  def testAsylum1Three(self):
+    self.char.lore_luck_slider = 2
+    self.state.event_stack.append(encounters.Asylum1(self.char))
+    with mock.patch.object(events.random,"randint", new=mock.MagicMock(return_value=5)):
```
Comment (fingoldfish) [inline eldritch/test_encounters.py:462]:
```suggestion
    with mock.patch.object(events.random, "randint", new=mock.MagicMock(return_value=5)):
```

File: eldritch/encounters.py [hunk 6]
```diff
@@ -136,10 +136,27 @@ def Administration7(char):
...
+  roll0 = events.Sequence([loss, gain1], char)
+  roll1 = events.Gain(char, {"clues": 2})
+  roll3 = events.Gain(char, {"clues": 3})
+  cond = events.Conditional(char, check, "successes", {0: roll0, 1: roll1, 3: roll3})
+  return events.Sequence([check, cond], char)
```
Comment (fingoldfish) [inline eldritch/encounters.py:148]: I know I have some style nitpicks later about two blank lines between top-level definitions, but I don't know that it makes a lot of sense here - pick any standard you want for the encounters file (okay, not any standard - 0, 1, or 2 lines between encounter definitions), and let's make the whole file consistent with the standard you pick.

**Review by fingoldfish — APPROVED** [review]

File: eldritch/encounters.py (hunk 2 above)
Comment (ThomasLipp) [reply eldritch/encounters.py]: Done

File: eldritch/test_encounters.py (hunk 3 above)
Comment (ThomasLipp) [reply eldritch/test_encounters.py]: done

File: eldritch/test_encounters.py (hunk 5 above)
Comment (ThomasLipp) [reply eldritch/test_encounters.py]: done
